src/irr/asset/CGLILoader.cpp

```diff
@@ -0,0 +1,552 @@
+				case gl::INTERNAL_RGBA32F: return getTranslatedFinalFormat(EF_R32G32B32A32_SFLOAT);			//GL_RGBA32F
+
+				case gl::INTERNAL_R64F_EXT: return getTranslatedFinalFormat(EF_R64_SFLOAT);			//GL_R64F
+				case gl::INTERNAL_RG64F_EXT: return getTranslatedFinalFormat(EF_R64G64_SFLOAT);		//GL_RG64F
+				///case gl::INTERNAL_RGB64F_EXT: return getTranslatedFinalFormat(EF_R64G64B64_SFLOAT);		//GL_RGB64F
+				case gl::INTERNAL_RGBA64F_EXT: return getTranslatedFinalFormat(EF_R64G64B64A64_SFLOAT);		//GL_RGBA64F
+
+				// sRGB formats
+				case gl::INTERNAL_SR8: return getTranslatedFinalFormat(EF_R8_SRGB);				//GL_SR8_EXT
+				case gl::INTERNAL_SRG8: return getTranslatedFinalFormat(EF_R8G8_SRGB);				//GL_SRG8_EXT
+				case gl::INTERNAL_SRGB8: return getTranslatedFinalFormat(EF_R8G8B8_SRGB);			//GL_SRGB8
+				case gl::INTERNAL_SRGB8_ALPHA8: return getTranslatedFinalFormat(EF_R8G8B8A8_SRGB);		//GL_SRGB8_ALPHA8
+
+				// Packed formats
+				case gl::INTERNAL_RGB9E5: return getTranslatedFinalFormat();			//GL_RGB9_E5
+				case gl::INTERNAL_RG11B10F: return getTranslatedFinalFormat();			//GL_R11F_G11F_B10F
+				case gl::INTERNAL_RG3B2: return getTranslatedFinalFormat();			//GL_R3_G3_B2
+				case gl::INTERNAL_R5G6B5: return getTranslatedFinalFormat();			//GL_RGB565
+				case gl::INTERNAL_RGB5A1: return getTranslatedFinalFormat();			//GL_RGB5_A1
+				case gl::INTERNAL_RGBA4: return getTranslatedFinalFormat();			//GL_RGBA4
+
+				case gl::INTERNAL_RG4_EXT: return getTranslatedFinalFormat();
+
+				// Luminance Alpha formats
+				case gl::INTERNAL_LA4: return getTranslatedFinalFormat();				//GL_LUMINANCE4_ALPHA4
+				case gl::INTERNAL_L8: return getTranslatedFinalFormat();				//GL_LUMINANCE8
+				case gl::INTERNAL_A8: return getTranslatedFinalFormat();				//GL_ALPHA8
+				case gl::INTERNAL_LA8: return getTranslatedFinalFormat();				//GL_LUMINANCE8_ALPHA8
+				case gl::INTERNAL_L16: return getTranslatedFinalFormat();				//GL_LUMINANCE16
+				case gl::INTERNAL_A16: return getTranslatedFinalFormat();				//GL_ALPHA16
+				case gl::INTERNAL_LA16: return getTranslatedFinalFormat();				//GL_LUMINANCE16_ALPHA16
+
+				// Depth formats
+				case gl::INTERNAL_D16: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT16
+				case gl::INTERNAL_D24: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT24
+				case gl::INTERNAL_D16S8_EXT: return getTranslatedFinalFormat();
+				case gl::INTERNAL_D24S8: return getTranslatedFinalFormat();			//GL_DEPTH24_STENCIL8
+				case gl::INTERNAL_D32: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT32
+				case gl::INTERNAL_D32F: return getTranslatedFinalFormat();				//GL_DEPTH_COMPONENT32F
+				case gl::INTERNAL_D32FS8X24: return getTranslatedFinalFormat();		//GL_DEPTH32F_STENCIL8
+				case gl::INTERNAL_S8_EXT: return getTranslatedFinalFormat();			//GL_STENCIL_INDEX8
+
+				// Compressed formats
+				case gl::INTERNAL_RGB_DXT1: return getTranslatedFinalFormat();						//GL_COMPRESSED_RGB_S3TC_DXT1_EXT
```

Comment:
https://github.com/buildaworldnet/IrrlichtBAW/blob/master/include/irr/asset/format/EFormat.h#L152

Reply:
but there is a DXT, and our prefix is BC(X). Is it the same?

Reply:
DXT1-5 and BC1-3 are different names for the same thing, DXT2 and DXT4 are not even formats in their own right.

Reply:
I will study it